sim/paladin/soc.go

```diff
@@ -98,8 +98,8 @@ func (paladin *Paladin) registerSealOfCommand() {
 			ActionID:    core.ActionID{SpellID: rank.proc.spellID},
 			SpellSchool: core.SpellSchoolHoly,
 			DefenseType: core.DefenseTypeMelee,
-			ProcMask:    core.ProcMaskMeleeMHSpecial | core.ProcMaskProc,
-			Flags:       core.SpellFlagMeleeMetrics | SpellFlag_RV, // RV Worked on PTR
+			ProcMask:    core.ProcMaskMeleeMHSpecial | core.ProcMaskMeleeProc | core.ProcMaskMeleeDamageProc,
```

Comment:
Does SoC need both the normal melee proc mask and the melee damage proc mask or could it work with just the damage one?

Reply:
it doesn't need it currently, but I'm implementing it this way so that it procs things that require damage in a melee, as well as those that only require it to be a melee proc.  I think a damaging melee proc should be flagged as a damage melee proc and a melee proc.